indico/modules/events/tracks/client/js/index.js

```diff
@@ -32,36 +60,12 @@
           placeholder: 'track-placeholder',
           connectWith: '.track-list',
           update: function(_event, ui) {
-            if (!ui.sender) {
-              // only trigger once when moving from one sortable to another
-              const $this = $(this);
-              const sortedList = $this
-                .find('li.track-row')
-                .map(function() {
-                  const $this = $(this);
-                  if ($this.hasClass('track-group-box')) {
-                    return {id: $this.data('id'), type: 'group'};
-                  } else {
-                    let parent = null;
-                    const parentDiv = $this.closest('.track-group-box');
-                    if (parentDiv.length) {
-                      parent = parentDiv.data('id');
-                    }
-                    return {id: $this.data('id'), type: 'track', parent};
-                  }
-                })
-                .get();
-              $.ajax({
-                url: $this.data('url'),
-                method: 'POST',
-                contentType: 'application/json',
-                data: JSON.stringify({sort_order: sortedList}),
-                complete: IndicoUI.Dialogs.Util.progress(),
-                error: handleAjaxError,
-              });
+            // call update only once and only for the receiver
+            if (this === ui.item.parent()[0]) {
+              updateSorting($('#track-list-container'));
             }
           },
-          receive: function(event, ui) {
+          receive: function(_event, ui) {
```

Comment:
why the `_event`?

Reply:
Hm, I think that was done accidentally by the ide because event is an unused variable and I decided to keep it.

Reply:
i'd rather use use `_` without anything after it in that case